heat/core/tests/test_tensor.py

```diff
@@ -735,6 +737,40 @@ def test_empty_like(self):
         with self.assertRaises(TypeError):
             ht.empty_like(ones, split='axis')
 
+    def test_right_hand_side_operations(self):
+        """
+        This test ensures that for each arithmetic operation (e.g. +, -, *, ...) that is implemented in the tensor
+        class, it works both ways.
+
+        Examples
+        --------
+        >>> import heat as ht
+        >>> T = ht.float32([[1., 2.], [3., 4.]])
+        >>> assert T * 3 == 3 * T
+        
+        """
+        operators = (
+            ('__add__', operator.add, True),
+            ('__sub__', operator.sub, False),
+            ('__mul__', operator.mul, True),
+            ('__truediv__', operator.truediv, False),
+            ('__floordiv__', operator.floordiv, False),
+            ('__mod__', operator.mod, False),
+            ('__pow__', operator.pow, False)
+        )
+        tensor = ht.float32([[1, 4], [2, 3]])
```

Comment:
We need test cases here for split tensors

Reply:
This is related to #167, which needs to be fixed first
At the moment, binary operations do not work properly for split tensors.